corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -45,7 +45,10 @@ var caseType = function (
     self.fetchCaseProperties = function () {
         if (self.groups().length === 0) {
             const caseTypeUrl = self.dataUrl + self.name + '/';
-            recurseChunks(caseTypeUrl).then(() => self.groups.sort(sortGroupsFn));
+            recurseChunks(caseTypeUrl).then(() => {
+                self.groups.sort(sortGroupsFn);
+                self.resetSaveButton();
```

Comment:
Out of curiosity is this fixing another bug?

Reply:
Yes! I could have specified in the commit message. Without this, sorting the `groups` observable is seen as a change and the "Save" button is re-enabled, as well as warning the user when they try to navigate away from the page. This is the same reason `self.resetSaveButton()` is called here, after `setCaseProperties` modifies `groups`: https://github.com/dimagi/commcare-hq/blob/036b6db21705a6e6220863d2408687652528fc87/corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js#L57-L58